packages/fxa-auth-server/lib/senders/emails/global.scss

```diff
@@ -143,9 +152,18 @@ tr > td:first-child {
   @extend .mt-2;
 }
 
-.large-code div {
+%code-common {
   text-align: center !important;
   font-family: monospace;
-  @extend .text-3xl;
   @extend .mb-6;
 }
+
+.large-code div {
```

Comment:
We could call this `code-large` and `code-medium` instead to follow our other conventions (and then you can nest these in a `.code { &-small ...` css set if you want)